locales/hu/messages.json

```diff
@@ -0,0 +1,128 @@
+      }
+    }
+  },
+  "increasePropertyValue": {
+    "message": "$PROPERTY$ értékének növelése",
+    "placeholders": {
+      "property": {
+        "content": "$1"
+      }
+    }
+  },
+  "resetPropertyValue": {
+    "message": "$PROPERTY$ értékének visszaállítása",
+    "placeholders": {
+      "property": {
+        "content": "$1"
+      }
+    }
+  },
+  "propertiesWereEnabledDisabled": {
+    "message": "Az egyedi szövegtérköz-tulajdonságok $STATUS$.",
+    "placeholders": {
+      "status": {
+        "content": "$1"
+      }
+    }
+  },
+  "enabled": {
+    "message": "engedélyezve lettek"
+  },
+  "disabled": {
+    "message": "le lettek tiltva"
+  },
+  "propertiesWereSetToWCAG": {
+    "message": "Az egyedi szövegtérköz-tulajdonságok a WCAG értékekre lettek beállítva."
+  },
+  "propertiesWereResetToDefaults": {
+    "message": "Az egyedi szövegtérköz-tulajdonságok az alapértelmezett értékeikre lettek visszaállítva."
+  },
+  "propertyWasResetToDefault": {
+    "message": "Az egyedi $PROPERTY$ tulajdonság az alapértelmezett értékére lett visszaállítva.",
+    "placeholders": {
+      "property": {
+        "content": "$1"
+      }
+    }
+  },
+  "instructionsForUsage": {
+    "message": "Hogyan használja a Szövegtérköz-szerkesztőt?"
+  },
+  "instructionsForActive": {
+    "message": "A felül lévő <strong class='hint'>Aktív</strong> kapcsoló segítségével aktiválhatja, illetve kikapcsolhatja az Ön által beállított egyedi szövegtérköz-tulajdonságokat, valamint a bővítmény interaktív vezérlőit. Ugyanehhez a funkcióhoz a <code class='hint'>Ctrl+Shift+F</code> (Mac-en a <code class='hint'>Command+Shift+F</code>) billentyűkombinációt is használhatja. (Igény szerint egyéni billentyűkombináció is megadható a böngésző bővítmény beállításaiban.) Az egyéni tulajdonságok átkapcsolásának lehetősége praktikus az egyéni beállítások teszteléséhez és tárolásához."
+  },
+  "instructionsForWCAGButton": {
+    "message": "A <strong class='hint'>WCAG értékek beállítása</strong> gomb segítségével az összes egyéni szövegtérköz-tulajdonságot a <a href='#wcag-conformance-values'>WCAG megfeleléshez szükséges értékekre</a> állíthatja be."
+  },
+  "instructionsForResetAllButton": {
+    "message": "Az <strong class='hint'>Összes érték visszaállítása</strong> gomb segítségével az összes szövegtérköz-tulajdonságot visszaállíthatja az alapértelmezett értékére."
+  },
+  "instructionsForSliders": {
+    "message": "Az egyes szövegtérköz-tulajdonságok értékeinek beállításához használja a <strong class='hint'>csúszkákat</strong>, vagy a <strong class='hint'><span class='visually-hidden'>Csökkentés</span><svg class='icon icon--inline' viewBox='0 0 24 24' ><path d='M19 13H5v-2h14v2z'></path></svg></strong> és <strong class='hint'><span class='visually-hidden'>Növelés</span><svg class='icon icon--inline' viewBox='0 0 24 24'><path d='M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z'></path></svg></strong> gombokat (amelyek támogatják a nyomva tartást is)."
```

Comment:
At the end of this string, the last part of the translation is missing after the "(that support long-press event)" text: "to set the values of the individual text spacing properties."

Reply:
There is no missing text. I have reversed the order of the sentence, so that the quoted text is at the beginning of the sentence.